vector/src/main/java/im/vector/activity/WidgetActivity.kt

```diff
@@ -65,7 +65,11 @@ class WidgetActivity : VectorAppCompatActivity() {
                         //already there
                     } else {
                         RoomWidgetPermissionBottomSheet
-                                .newInstance(viewModel.session!!.myUserId, viewModel.widget)
+                                .newInstance(viewModel.session!!.myUserId, viewModel.widget).apply {
```

Comment:
Does this code is robust to screen rotation, will the bottomSheet is displayed?

Reply:
worst case you have to tap again on the video conf banner